control-plane-operator/controllers/hostedcontrolplane/kas/kms/ibmcloud.go

```diff
@@ -58,7 +58,7 @@ func NewIBMCloudKMSProvider(ibmCloud *hyperv1.IBMCloudKMSSpec, kmsImage string)
 	}, nil
 }
 
-func (p *ibmCloudKMSProvider) GenerateKMSEncryptionConfig() (*v1.EncryptionConfiguration, error) {
+func (p *ibmCloudKMSProvider) GenerateKMSEncryptionConfig(apiVersion string) (*v1.EncryptionConfiguration, error) {
```

Comment:
```suggestion
func (p *ibmCloudKMSProvider) GenerateKMSEncryptionConfig(_ string) (*v1.EncryptionConfiguration, error) {
```

Reply:
good catch! I thought I had already replaced it, but I must have missed it.